src/main/java/com/heeverse/concert/service/ConcertService.java

```diff
@@ -27,8 +34,17 @@ public void registerConcert(List<ConcertRequestDto> listDto) {
             Long concertSeq = concertMapper.insertConcert(new Concert(dto));
 
             TicketRequestDto ticketRequestDto = new TicketRequestDto(concertSeq,
-                dto.getTicketGradeDtoList());
+                dto.getConcertDate(), dto.getTicketGradeDtoList());
             ticketService.registerTicket(ticketRequestDto);
         }
     }
+
+    @Transactional(readOnly = true)
+    public List<SearchConcertResponseDto> getConcertList(SearchConcertRequestDto dto) {
+
+        List<SearchConcertDto> concertList = concertMapper.selectConcertList(dto);
+        return concertList.stream().map(d -> new SearchConcertResponseDto(d.getConcertName(),
+                d.getConcertDate(), d.getArtistNameKor(), d.getArtistNameEng(), d.getVenueName()))
+            .collect(Collectors.toList());
```

Comment:
DTO를 그대로 생성자에 넘겨주는것도 좋은 방법입니다. 코드가 훨씬 간결해지고 생성자에서 요구하는 DTO의 스펙도 더 명확하게 처리할 수 있어요.

```suggestion
        return concertList.stream().map(SearchConcertResponseDto::new)
            .collect(Collectors.toList());
```